src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -269,17 +269,17 @@ const terminalConfiguration: IConfigurationNode = {
 		[TerminalSettingId.TerminalTitleSeparator]: {
 			'type': 'string',
 			'default': ' - ',
-			'markdownDescription': localize("terminal.integrated.tabs.separator", "Separator used by `terminal.integrated.title` and `terminal.integrated.description`.")
+			'markdownDescription': localize("terminal.integrated.tabs.separator", "Separator used by `terminal.integrated.tabs.title` and `terminal.integrated.tabs.description`.")
```

Comment:
Even better to avoid translating the settings ids and markdown `:

```suggestion
			'markdownDescription': localize("terminal.integrated.tabs.separator", "Separator used by {0} and {0}.", `\`${TerminalSettingId.TerminalTitle}\``, `\`${TerminalSettingId.TerminalDescription}\``)
```